crates/astria-sequencer-validation/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "astria-sequencer-validation"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+ct-merkle = "0.1"
+
+bytes = { workspace = true }
+eyre = { workspace = true }
+hex = { workspace = true, features = ["serde"] }
+serde = { workspace = true, features = ["derive"] }
+sha2 = { workspace = true }
+tendermint = { workspace = true }
```

Comment:
Noted in the crate. I think this should not have a direct dep on tendermint. Alternatively, the methods that do use tendermint hashes should have a `*tendermint*` or something like that in their name because it's very surprising to find them there.

I do think however that we don't want such a direct dependency.